### PR TITLE
Update the find corpus tool to provide more information

### PR DESCRIPTION
**Pull request by gregtatum**
I was finding the dataset acquisition step a bit confusing while working on opus cleaner, so I enhanced the script that finds datasets for us. Previously this tool just outputted the YAML keys. Now it provides a table of information. Note that the commits are well-ordered if you want to review it that way.

```
Fetching datasets from:
https://opus.nlpl.eu/opusapi/?source=en&target=ca&preprocessing=moses&version=latest


┌──────────────────────────────┐
│ OPUS - https://opus.nlpl.eu/ │
└──────────────────────────────┘

Dataset                        Code                                   Sentences Size     URL
────────────────────────────── ────────────────────────────────────── ───────── ──────── ──────────────────────────────────────────────────────────
CCMatrix                       opus_CCMatrix/v1                       21284430  1.6 GB   https://opus.nlpl.eu/CCMatrix-v1.php
NLLB                           opus_NLLB/v1                           21284430  1.6 GB   https://opus.nlpl.eu/NLLB-v1.php
HPLT                           opus_HPLT/v1.1                         8905979   825.4 MB https://opus.nlpl.eu/HPLT-v1.1.php
MultiHPLT                      opus_MultiHPLT/v1.1                    8905890   838.0 MB https://opus.nlpl.eu/MultiHPLT-v1.1.php
CCAligned                      opus_CCAligned/v1                      5802549   535.4 MB https://opus.nlpl.eu/CCAligned-v1.php
MaCoCu                         opus_MaCoCu/v2                         3130523   388.1 MB https://opus.nlpl.eu/MaCoCu-v2.php
MultiMaCoCu                    opus_MultiMaCoCu/v2                    3130520   389.5 MB https://opus.nlpl.eu/MultiMaCoCu-v2.php
XLEnt                          opus_XLEnt/v1.2                        3000401   79.6 MB  https://opus.nlpl.eu/XLEnt-v1.2.php
WikiMatrix                     opus_WikiMatrix/v1                     1205909   156.2 MB https://opus.nlpl.eu/WikiMatrix-v1.php
OpenSubtitles                  opus_OpenSubtitles/v2018               482598    14.8 MB  https://opus.nlpl.eu/OpenSubtitles-v2018.php
wikimedia                      opus_wikimedia/v20230407               398889    53.2 MB  https://opus.nlpl.eu/wikimedia-v20230407.php
KDE4                           opus_KDE4/v2                           169604    5.4 MB   https://opus.nlpl.eu/KDE4-v2.php
QED                            opus_QED/v2.0a                         73936     4.8 MB   https://opus.nlpl.eu/QED-v2.0a.php
TED2020                        opus_TED2020/v1                        52696     3.7 MB   https://opus.nlpl.eu/TED2020-v1.php
ELRC-5129-GEnCaTA_a            opus_ELRC-5129-GEnCaTA_a/v1            38596     3.8 MB   https://opus.nlpl.eu/ELRC-5129-GEnCaTA_a-v1.php
GNOME                          opus_GNOME/v1                          12962     455.7 kB https://opus.nlpl.eu/GNOME-v1.php
GlobalVoices                   opus_GlobalVoices/v2018q4              7226      705.5 kB https://opus.nlpl.eu/GlobalVoices-v2018q4.php
Books                          opus_Books/v1                          4605      335.9 kB https://opus.nlpl.eu/Books-v1.php
EUbookshop                     opus_EUbookshop/v2                     4205      355.3 kB https://opus.nlpl.eu/EUbookshop-v2.php
ELRC-5189-Catalan_WMT2013_Mach opus_ELRC-5189-Catalan_WMT2013_Mach/v1 3001      283.6 kB https://opus.nlpl.eu/ELRC-5189-Catalan_WMT2013_Mach-v1.php
Tatoeba                        opus_Tatoeba/v2023-04-12               1892      75.8 kB  https://opus.nlpl.eu/Tatoeba-v2023-04-12.php
ELRC-3040-wikipedia_health     opus_ELRC-3040-wikipedia_health/v1     1532      167.9 kB https://opus.nlpl.eu/ELRC-3040-wikipedia_health-v1.php
ELRC-wikipedia_health          opus_ELRC-wikipedia_health/v1          1532      169.0 kB https://opus.nlpl.eu/ELRC-wikipedia_health-v1.php
ELRC_2922                      opus_ELRC_2922/v1                      1531      173.1 kB https://opus.nlpl.eu/ELRC_2922-v1.php
ELRC-5190-Cyber_MT_Test        opus_ELRC-5190-Cyber_MT_Test/v1        1019      139.3 kB https://opus.nlpl.eu/ELRC-5190-Cyber_MT_Test-v1.php
tldr-pages                     opus_tldr-pages/v2023-08-29            937       25.6 kB  https://opus.nlpl.eu/tldr-pages-v2023-08-29.php
ELRC-Museus_2007               opus_ELRC-Museus_2007/v1               125       7.2 kB   https://opus.nlpl.eu/ELRC-Museus_2007-v1.php
ELRC-Localidades_2007          opus_ELRC-Localidades_2007/v1          101       8.2 kB   https://opus.nlpl.eu/ELRC-Localidades_2007-v1.php
ELRC-2638-monumentos_2007      opus_ELRC-2638-monumentos_2007/v1      17        8.2 kB   https://opus.nlpl.eu/ELRC-2638-monumentos_2007-v1.php
ELRC-2614-Localidades_2007     opus_ELRC-2614-Localidades_2007/v1     10        8.2 kB   https://opus.nlpl.eu/ELRC-2614-Localidades_2007-v1.php
ELRC-2612-Artigos_visitportuga opus_ELRC-2612-Artigos_visitportuga/v1 9         7.2 kB   https://opus.nlpl.eu/ELRC-2612-Artigos_visitportuga-v1.php
ELRC-2616-Museus_2007          opus_ELRC-2616-Museus_2007/v1          8         7.2 kB   https://opus.nlpl.eu/ELRC-2616-Museus_2007-v1.php
ELRC-Artigos_visitportuga      opus_ELRC-Artigos_visitportuga/v1      6         7.2 kB   https://opus.nlpl.eu/ELRC-Artigos_visitportuga-v1.php
ELRC-2480-Estatuto_dos_Deputad opus_ELRC-2480-Estatuto_dos_Deputad/v1 2         1.0 kB   https://opus.nlpl.eu/ELRC-2480-Estatuto_dos_Deputad-v1.php
ELRC-2481-Constituio_da_Repbli opus_ELRC-2481-Constituio_da_Repbli/v1 2         1.0 kB   https://opus.nlpl.eu/ELRC-2481-Constituio_da_Repbli-v1.php
Ubuntu                         opus_Ubuntu/v14.10                               175.1 kB https://opus.nlpl.eu/Ubuntu-v14.10.php

YAML:
    - opus_Books/v1
    - opus_CCAligned/v1
    - opus_CCMatrix/v1
    - opus_ELRC-2480-Estatuto_dos_Deputad/v1
    - opus_ELRC-2481-Constituio_da_Repbli/v1
    - opus_ELRC-2612-Artigos_visitportuga/v1
    - opus_ELRC-2614-Localidades_2007/v1
    - opus_ELRC-2616-Museus_2007/v1
    - opus_ELRC-2638-monumentos_2007/v1
    - opus_ELRC-3040-wikipedia_health/v1
    - opus_ELRC-5129-GEnCaTA_a/v1
    - opus_ELRC-5189-Catalan_WMT2013_Mach/v1
    - opus_ELRC-5190-Cyber_MT_Test/v1
    - opus_ELRC-Artigos_visitportuga/v1
    - opus_ELRC-Localidades_2007/v1
    - opus_ELRC-Museus_2007/v1
    - opus_ELRC-wikipedia_health/v1
    - opus_ELRC_2922/v1
    - opus_EUbookshop/v2
    - opus_GNOME/v1
    - opus_GlobalVoices/v2018q4
    - opus_HPLT/v1.1
    - opus_KDE4/v2
    - opus_MaCoCu/v2
    - opus_MultiHPLT/v1.1
    - opus_MultiMaCoCu/v2
    - opus_NLLB/v1
    - opus_OpenSubtitles/v2018
    - opus_QED/v2.0a
    - opus_TED2020/v1
    - opus_Tatoeba/v2023-04-12
    - opus_Ubuntu/v14.10
    - opus_WikiMatrix/v1
    - opus_XLEnt/v1.2
    - opus_tldr-pages/v2023-08-29
    - opus_wikimedia/v20230407

┌─────────────────────────────────────────────────┐
│ sacrebleu - https://github.com/mjpost/sacrebleu │
└─────────────────────────────────────────────────┘

Dataset Description URLs
─────── ─────────── ────
(no datasets)

YAML:
(no datasets)


┌────────────────────────────────────────────────┐
│ mtdata - https://github.com/thammegowda/mtdata │
└────────────────────────────────────────────────┘

Dataset                                URL
────────────────────────────────────── ───────────────────────────────────────────────────────────────────────────────────────────────────────────
mtdata_ELRC-wikipedia_health-1-cat-eng https://elrc-share.eu/repository/download/ac6d557e8de811ea913100155d026706b0c5fee96b88489781ddd7675f8ea2ae/
mtdata_Facebook-wikimatrix-1-cat-eng   https://dl.fbaipublicfiles.com/laser/WikiMatrix/v1/WikiMatrix.ca-en.tsv.gz
mtdata_Statmt-ccaligned-1-cat_ES-eng   http://www.statmt.org/cc-aligned/sentence-aligned/ca_ES-en_XX.tsv.xz

YAML:
    - mtdata_ELRC-wikipedia_health-1-cat-eng
    - mtdata_Facebook-wikimatrix-1-cat-eng
    - mtdata_Statmt-ccaligned-1-cat_ES-eng
```

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -136,3 +136,5 @@ dmypy.json
 .models
 .bin
 .snakemake
+
+tests_data
```
Comment (gregtatum) [inline .gitignore:140]: This is a drive-by fix, as #219 creates this directory during testing.

**Review by nordzilla — APPROVED** [review]
@gregtatum 

I read through all of this and everything seems reasonable to me. 

The output seems nice.

I don't have a lot to comment on regarding Pythonic idioms or anything like that, because I'm not much of an expert with Python myself.

File: utils/find_corpus.py [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+    # The language tag.
+    target: str
+    # The URL to the download
+    url: str
+    # For example "v1"
+    version: str
+
+    alignment_pairs: int
+    id: int
+    # Size in KiB?
```
[the remark this answers is not in the document]
Comment (gregtatum) [reply utils/find_corpus.py:39]: I just downloaded one and confirmed it's KiB by downloading a zip and dividing it by 1024.

**Comment by gregtatum** [comment]
My force push is a rebase, and removing a ?.
